samples/BlazorIdentity.ServerApp/wwwroot/js/Alert.js

```diff
@@ -0,0 +1,14 @@
+﻿function DismissAlert(divName) {
+    var mydiv = document.getElementById(divName);
+    mydiv.setAttribute("hidden", "true")
+    mydiv.hidden = "true";
+    document.getElementById(divName).hidden = true;
+}
+
+function ShowAlert(divName) {
+//    var mydiv = document.getElementById(divName).hidden = false;
+//    mydiv.removeAttribute("hidden");
+//    mydiv.hidden = "false";
+//    mydiv.setAttribute("hidden", "false")
+    document.getElementById(divName).hidden = false;
+}
```

Comment:
There should be no custom JS in the sample app, any JS required to support the functionality should be in one of the library projects and invoked by the components or classes via JS interop.

As for alerting in general, I'd leave it up to the hosting page to manage that aspect of the UI and in the sample Blazor Server app I'm already using the BlazorStrap components to do that, e.g. https://github.com/DamianEdwards/BlazorIdentity/blob/afefa6a96012e5245b6f8957b60d7f658b7a4345/samples/BlazorIdentity.ServerApp/Pages/Account/Manage/Index.razor